src/platform_impl/windows/event_loop.rs

```diff
@@ -2153,9 +2201,10 @@ unsafe extern "system" fn thread_event_target_callback<T: 'static>(
                         }
                     }
 
-                    if util::has_flag(mouse.usButtonFlags, winuser::RI_MOUSE_WHEEL) {
-                        let delta =
-                            mouse.usButtonData as SHORT as f32 / winuser::WHEEL_DELTA as f32;
+                    let mouse_button_flags = mouse.Anonymous.Anonymous.usButtonFlags;
+
+                    if util::has_flag(mouse_button_flags as u32, RI_MOUSE_WHEEL) {
```

Comment:
There's this nice `has_flag` function, can we preplace the current `& ==` pairs with it? Should make it easier to find-replace with `.contains()` if/when Kenny gets around to adding that to windows-rs (assuming it doesn't add too much compile-time).